src/lightbox/downloadFileToCache.js

```diff
@@ -0,0 +1,21 @@
+/* @flow strict-local */
+import RNFetchBlob from 'rn-fetch-blob';
+import { getMimeTypeFromFileExtension } from '../utils/url';
+
+/**
+ * Download a remote file to the app's cache directory.
+ *
+ * @param tempUrl A URL to the file. Should be a valid temporary URL generated
+ *     using `getFileTemporaryUrl`.
+ * @param fileName Name of the file to be downloaded. Should include the
+ *     extension.
+ */
+
+export default async (tempUrl: string, fileName: string): Promise<mixed> =>
```

Comment:
nit: no blank line between jsdoc and the thing it's documenting -- that helps keep it visually clear what it's attached to